packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -457,6 +460,15 @@ export class ARRenderer extends EventDispatcher {
       this.dispatchEvent({type: 'status', status: ARStatus.SESSION_STARTED});
     }
 
+    // Use automatic dynamic viewport scaling if supported.
+    if (view.requestViewportScale && view.recommendedViewportScale) {
+      const scale = view.recommendedViewportScale;
+      view.requestViewportScale(Math.max(scale, MIN_VIEWPORT_SCALE));
+    }
+    const layer = this[$currentSession]!.renderState.baseLayer;
+    const viewport = layer!.getViewport(view);
```

Comment:
Our tests are failing on this line in Chrome 85. Any idea why?

Reply:
The mock XRSession data was missing the getViewport method on the layer.

Is there a way to run tests individually? My laptop seems to have trouble running the full suite, so the fix is a bit speculative.

Reply:
Yes, on any of the `test()` or `suite()` calls, just add a `.only`, like `test.only()`.

Reply:
Anyway, it passed our CI, so looks like it's all good. Thanks!